Doc/library/contextlib.rst

```diff
@@ -314,7 +314,9 @@ Functions and classes provided:
 
    If the code within the :keyword:`!with` block raises a
    :exc:`BaseExceptionGroup`, suppressed exceptions are removed from the
-   group.  If any exceptions in the group are not suppressed, a group containing them is re-raised.
+   group.  If any exceptions in the group are not suppressed, a group containing
+   them is created using the old group's :meth:`~BaseExceptionGroup.derive`
+   method, and the new group is then re-raised.
```

Comment:
```suggestion
   group.  Any exceptions of the group which are not suppressed are re-raised in
   a new group which is created by the original group's :meth:`~BaseExceptionGroup.derive`
   method.
```

How about that?

Reply:
Nice!

Reply:
Or "created using" rather than "created by"?